### PR TITLE
filtering and sorting a list of strings

### PR DESCRIPTION
**Pull request by davegomez**
## TL;DR;

You may have a big list of names you need to filter in order to remove duplicates and sort them alphabetically.
## davegomez

[@davegomez](https://twitter.com/davegomez)

### CODE REVIEW

File: _posts/en/2016-XX-XX-filtering-and-sorting-a-list-of-strings.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+
+title: Filtering and Sorting a List of Strings
+tip-number: 25
```
Comment (loverajoel) [inline _posts/en/2016-XX-XX-filtering-and-sorting-a-list-of-strings.md:5]: update to 26

**Comment by loverajoel** [comment]
update the file name to today

**Comment by loverajoel** [comment]
@davegomez ping!

**Comment by loverajoel** [comment]
Thanks man

**Comment by davegomez** [comment]
I'm sorry @loverajoel, I just saw the email. Too much work!

**Pull request merged**